.github/workflows/ci.yml

```
@@ -599,12 +602,9 @@ jobs:
                   --amount 1000000
           done
 
-          "$BIN_DIR/mc-consensus-tool" wait-for-quiet \
-              insecure-mc://localhost:3200/ \
-              insecure-mc://localhost:3201/ \
-              insecure-mc://localhost:3202/ \
-              insecure-mc://localhost:3203/ \
-              insecure-mc://localhost:3204/
+          echo "Waiting for quiet after minting"
+          POST_MINT_BLOCK_INDEX=$("$BIN_DIR/mc-consensus-tool" wait-for-quiet --beyond-block $PRE_MINT_BLOCK_INDEX)
+          echo "Done waiting, POST_MINT_BLOCK_INDEX = ${POST_MINT_BLOCK_INDEX}"
```

:wrench: Same as above
```suggestion
          POST_MINT_BLOCK_COUNT=$("$BIN_DIR/mc-consensus-tool" wait-for-quiet --beyond-block $PRE_MINT_BLOCK_COUNT)
          echo "Done waiting, POST_MINT_BLOCK_COUNT = ${POST_MINT_BLOCK_COUNT}"
```

---

[ignore me](https://www.youtube.com/watch?v=-egkmhWMLdg)  

(see previous comment